modules/ebi-metagenomics/deeptmhmm/meta.yml

```diff
@@ -0,0 +1,45 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/yaml-schema.json
+name: deeptmhmm
+description: A Deep Learning Model for Transmembrane Topology Prediction and Classification
+keywords:
+  - sort
```

Comment:
sort, example? I think you forgot the keyworks

Reply:
I blame the linting. Should have reminded me :P